specification/datalake-analytics/resource-manager/Microsoft.DataLakeAnalytics/2016-11-01/account.json

```diff
@@ -1598,6 +1598,113 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DataLakeAnalytics/locations/{location}/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "Account"
+        ],
+        "operationId": "Account_CheckNameAvailability",
```

Comment:
For this one I would like to leave as is for this PR because we currently have existing APIs for "Account_Get", "Account_Create", etc.  We will make the fix to "Accounts" in an upcoming API version, 2017-10-01, which has other breaking changes -- so we would like to bundle the breaking changes together.